src/views/LoginView/LoginView.js

```diff
@@ -1,25 +1,18 @@
-import styles from './LoginView.scss'
-import autobind from 'autobind-decorator'
-import cx from 'classnames'
 import React, { Component, PropTypes } from 'react'
-import { Button, Field, Form, Header, Input, Message } from 'stardust'
 import { connect } from 'react-redux'
-import { bindActionCreators } from 'redux'
-import { login } from 'services/session'
+import {
+  Grid,
+  Header,
+  Image,
+  Column,
+} from 'stardust'
 
-const mapStateToProps = (state) => ({
-  session: state.session,
-})
-
-const mapDispatchToProps = (dispatch) => ({
-  actions: bindActionCreators({
-    login,
-  }, dispatch),
-})
+import './LoginView.scss'
+import LoginForm from 'components/LoginForm/LoginForm'
+import logoUrl from 'assets/images/android-icon-192x192.png'
```

Comment:
Welcome to webpack!  One of Webpack's main jobs is to pass your imported assets (of any kind) to a loader.  It does this by regex match of the `from '...'` path.  In this case, the regex is for anything that ends in a `.png` or `.jpg`.  If the regex `test` is a hit, then the imported thing (in this case an image) will be passed through the `file-loader`.  Since all loaders end in `-loader` you only need to list the loader as `file`.

The role of the `file-loader` is to give you a generated url using the md5 hash file contents:

![image](https://cloud.githubusercontent.com/assets/5067638/12970232/dd5f548a-d040-11e5-8951-96a6c440bdfc.png)

We can / probably should pull these into a util function so the actual paths are encapsulated.  Though, I'd rather use font icons and not really have images.  This is a just another hack-it-together-for-now move.